### PR TITLE
70 default items

### PR DESCRIPTION
**Pull request by chanson02**
Add items that a user can use even if their stamina/magic are 0

### CODE REVIEW

File: dunwoody_disaster/Item.py [hunk 1]
```diff
@@ -74,7 +74,10 @@ def to_dict(self) -> dict:
...
     def __init__(self, name, magic, damage, magicCost, staminaCost):
         super().__init__(name)
-        self.stats = WeaponStats[name]
+        try:
+            self.stats = WeaponStats[name]
+        except KeyError:
+            self.stats = [0, 0, 0]
         self.magic = magic
         self.damage = damage
         self.magicReq = magicCost
```
Comment (chanson02) [inline dunwoody_disaster/Item.py:83]: There is a disconnect between these. The `self.stats` are shown on the arsenal, but the `self.magic / self.damage / etc` are being used for the actual fight calculations. I vote we get rid of self.stats.

Comment (chanson02) [reply dunwoody_disaster/Item.py:83]: I opened #57 for this issue.

File: dunwoody_disaster/views/action_selector.py [hunk 2]
```diff
@@ -85,7 +85,10 @@ def selectRandom(self):
             and self.character.curStamina >= w.staminaCost
         ]
 
-        chosen_weapon = random.choice(weapons)
+        if len(weapons) > 0:
+            chosen_weapon = random.choice(weapons)
```
Comment (MLHumphries) [inline dunwoody_disaster/views/action_selector.py:89]: Will this still allow enemies to use these default attacks if they do not need to use them? I feel like we wouldn't want that to happen if so.

Comment (chanson02) [reply dunwoody_disaster/views/action_selector.py:89]: This is saying that the enemy will only use the default item if they can't use anything else.

**Review by NoahBartell — APPROVED** [review]
lgtm

**Review by MLHumphries — APPROVED** [review]

**Pull request merged**